astroplan/plots/plots.py

```diff
@@ -0,0 +1,294 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import matplotlib.pyplot as plt
+import numpy as np
+import astropy.units as u
+import random
+
+# Originally authored by Jazmin Berlanga Medina
+# (jazmin.berlanga@gmail.com), with input from
+# Adrian Price-Whelan (email), Erik Tollerud (email),
+# Christoph Deil (email), Eric Jeschke (email) and Brett Morris (email).
+
+
+# Replace function call later with below.
+def plot_airmass(target, observer, time, ax=None, style_kwargs=None, dark_plot=False):
+    """
+    TODO:
+        1) Timezones?
+        2) Limit airmass <= 3.
+        3) Plot lines instead of points.
+        4) Legend should not be transparent.
+        5) Airmass equation? Should this be moved to a separate function?
+        6) dark_plot option
+
+    Returns an ax object with an airmass vs. time plot.
+
+    If an ax object is passed in, plots an additional airmass plot on top.
+    Otherwise, creates a new ax object with an airmass plot.
+
+    Parameters
+    ----------
+    target: `~astroplan.FixedTarget`
+        The celestial body of interest.
+
+    observer: `~astroplan`
+        The person, telescope, observatory, etc. doing the observing.
+
+    time : `~astropy.time.Time`
+        An collection of times to use in generating data to plot.
+        Can be scalar (len(time)=1) or contain multiple times.
```

Comment:
You need to describe better what happens in the scalar case.  This description could be in the main description part of the docstrings, though.